src/components/MyMessageStatus.tsx

```diff
@@ -0,0 +1,35 @@
+import { SendableMessage } from '@sendbird/chat/lib/__definition';
+import { SendingStatus } from '@sendbird/chat/message';
+import { Locale } from 'date-fns';
+
+import { DefaultSentTime } from './MessageComponent';
+import { Icon } from '../foundation/components/Icon';
+import { Loader } from '../foundation/components/Loader';
+import { CommonTheme } from '../theme';
+import { formatCreatedAtToAMPM } from '../utils/messageTimestamp';
+
+interface MyMessageStatusProps {
+  message: SendableMessage;
+  dateLocale: Locale;
+  theme: CommonTheme;
+}
+
+export default function MyMessageStatus(props: MyMessageStatusProps) {
+  const { message, dateLocale, theme } = props;
+  switch (message.sendingStatus) {
+    case SendingStatus.PENDING:
+      return (
+        <Loader size={16} className="sendbird-loader">
+          <Icon type={'spinner'} color={theme.bgColor.outgoingMessage} size={16} />
+        </Loader>
+      );
+    case SendingStatus.FAILED:
+      return (
+        <Loader size={16} className="sendbird-loader no-animation">
+          <Icon type={'error'} color={'error'} size={16} />
+        </Loader>
+      );
+    default:
+      return <DefaultSentTime>{formatCreatedAtToAMPM(message.createdAt, dateLocale)}</DefaultSentTime>;
+  }
+}
```

Comment:
요런식으로 접근을 해보실 수 있을 것 같습니다!

```suggestion
export default function MyMessageStatus(props: MyMessageStatusProps) {
  const { message, dateLocale, theme } = props;
  switch (message.sendingStatus) {
    case SendingStatus.PENDING:
      return (
        <Loader size={16} className={container}>
          <Icon type={'spinner'} color={theme.bgColor.outgoingMessage} size={16} />
        </Loader>
      );
    case SendingStatus.FAILED:
      return (
        <div className={container}>
          <Icon type={'error'} color={'error'} size={16} />
        </div>
      );
    default:
      return <DefaultSentTime>{formatCreatedAtToAMPM(message.createdAt, dateLocale)}</DefaultSentTime>;
  }
}

const container = css`
  margin-bottom: 2px;
`
```

Reply:
theme 를 props 로 받지 않고, 이 컴포넌트 안에서 바로 useTheme 호출해서 사용해도 좋을 것 같아요!

Reply:
```
const sendbirdLoader = css`
  margin-bottom: 2px;
  &.no-animation {
    animation: unset;
  }
`
```
no-animation 을 사용하고 싶은 경우도에도 css 로 되는 syntax 가 있을까요? 아니면 styled 의 props 를 사용하는 방법도 있습니다

Reply:
컨테이너에 Loader 를 사용하지 않으면 될 것 같습니다.

Reply:
커밋했는데 확인 부탁드립니다